DOC-1341: RS - Update LDAP docs to reflect changes (#1936)
RedisLabs/redislabs-docs: content/rs/security/ldap/cluster-based-ldap-authentication.md

]
---
{{<warning>}}

> As of v6.2.12, the features described in this article are [obsolete]({{< relref "/glossary/#obsolete" >}}); that is, they have been removed from the product.<br/><br/>  

Reviewer: ```suggestion
As of v6.2.12, the features described in this article are [obsolete]({{< relref "/glossary/#obsolete" >}}) and have been removed from the product.<br/><br/>  
```
